src/modules/users/consumers/PermissionsConsumer.spec.ts

```diff
@@ -14,11 +14,12 @@ import { getApp } from '../../../app.js'
 import { SINGLETON_CONFIG } from '../../../infrastructure/parentDiConfig.js'
 import type { PermissionsService } from '../services/PermissionsService.js'
 
+import { generateMonotonicUuid } from '@lokalise/id-utils'
```

Comment:
Is `@lokalise/id-utils` an existing dependency? I am not seeing it in the `package.json`

Reply:
Good catch, indeed. Added: https://github.com/lokalise/node-service-template/pull/683/commits/f722889acef2133f3e91e515301b1d8c0f5f3d7e